ui/src/Components/ProfilePage.jsx

```diff
@@ -112,39 +133,83 @@ export default class ProfilePage extends React.Component {
+                  </Alert>                    
+                  ) : <div/>}
+                <Alert className="push-right" color="success">
+                  APPLICATION STATUS: {this.state.profile.result.app_status}
+                </Alert>
+                <h4 className="push-left">{this.state.profile.result.first_name} {' '}
+                {this.state.profile.result.middle_name} {' '}
+                {this.state.profile.result.last_name}</h4>
+              <div>
+              </div>
+              <Table responsive>
                 <tbody>
                 <tr>
-                    <td>Email</td>
+                    <th>Email</th>
                     <td>{this.state.profile.result.email}</td>
                   </tr>
                   <tr>
-                    <td>Phone Number</td>
-                    <td>{this.state.profile.result.contact_num}</td>
+                    <th>Phone Number</th>
+                    {this.state.profile.result.contact_ext ? (
+                    <td>{this.state.profile.result.contact_num.substring(0,3)}-{this.state.profile.result.contact_num.substring(3,6)}-{this.state.profile.result.contact_num.substring(6,10)}  x  {this.state.profile.result.contact_ext}</td>
+                  ) : <td>{this.state.profile.result.contact_num}</td>}
+                  </tr>
+                  <tr>
+                    <th>Address</th>
+                    {this.state.profile.result.address ? (
+                        <td><div>{this.state.profile.result.address}</div>
+                        <div>{this.state.profile.result.city},  {this.state.profile.result.postal_code.substring(0,3)} {this.state.profile.result.postal_code.substring(3,6)}   {this.state.profile.result.province}</div>
+                        </td>
+                      ) : <td>Not Provided</td>}
+                  </tr>
+                  <tr>
+                    <th>Date of Birth (MM/DD/YYYY)</th>
+                    {this.state.profile.result.dob ? (
+                      <td>{this.state.profile.result.dob.substring(0,2)}/{this.state.profile.result.dob.substring(2,4)}/{this.state.profile.result.dob.substring(4,8)}</td>
+                      ) : <td>Not Provided</td>}
+                  </tr>
+
+
+                  {references}
+
+
+                  <tr>
+                    <th>Current Occupation</th>
+                    {this.state.profile.result.dob ? (
+                      <td>{this.state.profile.result.current_occup}</td>
+                      ) : <td>Not Provided</td>}
+                  </tr>
+                  <tr>
+                    <th>Canadian ID Provided?</th>
+                    {this.state.profile.result.canadian_id ? (
+                    <td>Yes</td>
+                  ) : <td>No</td>}
                   </tr>
                   <tr>
-                    <td>Extension</td>
-                    <td>{this.state.profile.result.contact_ext}</td>
+                    <th>Health Insurance Provided?</th>
```

Comment:
Great catch changing up the `<td>` -> `<th>`

Reply:
🙆‍♀️